listenbrainz/webserver/views/stats_api.py

```diff
@@ -635,7 +631,105 @@ def get_artist_map(user_name: str):
     })
 
 
-def _process_entity(stats, stats_range, offset, count, entity) -> Tuple[list, int]:
+@stats_api_bp.route("/sitewide/artists")
+@crossdomain()
+@ratelimit()
+def get_sitewide_artist():
+    """
+    Get sitewide top artists.
+
+
+    A sample response from the endpoint may look like::
+
+        {
+            "payload": {
+                "artists": {
+                    "April 2020": [
```

Comment:
We should probably consider making this more usable for programmers, maybe group by year first, and then month?
```
artists: {
    2020: {
         0: [blah, blah],
         1: [blah, blah],
    }
} 
```

and so on.